src/material-experimental/mdc-button/fab.scss

```diff
@@ -3,6 +3,20 @@
 
 @include mdc-fab-without-ripple($query: $mat-base-styles-query);
 
+.mat-mdc-fab, .mat-mdc-mini-fab {
+  // ::before content is used for the interaction highlight (focus, press, active, hover)
+  &::before {
+    content: '';
```

Comment:
How are we coloring this? If it's with an `rgba` color it'll be an issue in high contrast mode, because `rgba` colors are turned into solid colors by Windows. The workaround is to make it a solid color with an `opacity`.